src/ca/mcgill/cs/jetuml/views/StringViewer.java

```diff
@@ -135,20 +186,31 @@ public void draw(String pString, GraphicsContext pGraphics, Rectangle pRectangle
 		final VPos oldVPos = pGraphics.getTextBaseline();
 		final TextAlignment oldAlign = pGraphics.getTextAlign();
 		
+		boolean isHorizontallyCentered = aAlign == Alignment.TOP_CENTER || aAlign == Alignment.CENTER_CENTER || 
+				aAlign == Alignment.BOTTOM_CENTER;

```

Comment:
you can define methods on the enum for this, e.g., `isLeft`, `isHorizontallyCentered` and `isRight`, which might allow you to cleanly inline the variables.